### PR TITLE
RenderTarget: Add `count` option for MRT.

### PR DESCRIPTION
**Pull request by CodyJasonBennett**
Related issue: #21930

**Description**

Introduces a `count` parameter to `RenderTarget` options for MRT and deprecates `WebGLMultipleRenderTargets`.

> **Note**: This PR aims to smoothen the 2D render target API, I haven't enhanced 3D and array cases as #22089 has done.

/cc @RenaudRohlinger

### CODE REVIEW

File: docs/api/en/renderers/WebGLRenderTarget.html [hunk 1]
```diff
@@ -49,7 +49,8 @@ <h3>
 			<br />
 			[page:Boolean depthBuffer] - default is `true`. <br />
 			[page:Boolean stencilBuffer] - default is `false`.<br />
-			[page:Number samples] - default is `0`.<br /><br />
+			[page:Number samples] - default is `0`.<br />
+			[page:Number count] - The number of render targets. Default is `1`.<br /><br />
```
Comment (CodyJasonBennett) [inline docs/api/en/renderers/WebGLRenderTarget.html:53]: This was copied and pasted from the MultipleRenderTargets docs, but looks out of context here. Is there any way to specify that this configures the number of color attachments without getting too technical?

**Comment by RenaudRohlinger** [comment]
Just reviewed it, looks good to me! Thanks for this Cody. Hopefully this time we can finally merge this one 👍

**Comment by Mugen87** [comment]
@mrdoob Before I review this, is removing `WebGLMultipleRenderTargets` and merging the functionality into `WebGLRenderTarget` okay with you? It is essentially the same approach like when we have removed `WebGLMultisampledRenderTarget`.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
Based on the general support of the PR and the positive signal of @mrdoob in https://github.com/mrdoob/three.js/pull/23444#issuecomment-1035010229 let's merge the PR!

**Pull request merged**